src/elf/note.rs

```diff
@@ -188,6 +188,15 @@ if_alloc! {
 
     impl<'a> Note<'a> {
         pub fn type_to_str(&self) -> &'static str {
+            // If the Note is from a coredump, match to different tags
+            if self.name == "CORE" {
```

Comment:
i haven't looked at this code in a while, but is it reliable to depend on self.name == "CORE" to parse from a coredump? E.g., are there cases when the file is a coredump, but self.name is not "CORE"?

Reply:
Afaik, looking at the linux source code, this should always be the case for the constants we care about:

https://elixir.bootlin.com/linux/v6.15.1/source/include/uapi/linux/elf.h#L399

Reply:
ack